### ISSUE TITLE
Using Scroll and Order in the same query returns bad results. 

### ISSUE THREAD

**Issue by tjoneslo** [Universal-Omega/DynamicPageList3#49]
https://gitlab.com/hydrawiki/extensions/DynamicPageList/-/issues/33

This is an old issue which I've rediscovered the hard way with the current version of the DPL

**Comment by tjoneslo** [comment]
Have the source of this problem:

https://github.com/Universal-Omega/DynamicPageList3/blob/REL1_35/includes/Parameters.php#L933

```
$titlegt = $wgRequest->getVal('DPL_fromTitle', '');
```
The default value of '' to this call is the source of the problem.  In theory you want this to be a space ' ', but the later code transforms this into an underscore '_'. The alternative is to use '!', which should be before any title in any wiki.

The end query is also broken:
https://github.com/Universal-Omega/DynamicPageList3/blob/REL1_35/includes/Parameters.php#L939

```
$titlelt = $wgRequest->getVal('DPL_toTitle', '');
```

Again, the default value of '' is the source of the problem. Finding the correct "last" character is an interesting problem. Using '~' works correctly for ASCII / Latin 1, but not for any searches beyond that. I'm not sure enough about configuration of PHP and MySQL version of UTF-8 to figure out what the good "last" character encompass potentially all of the possible article titles using non-ascii characters as a default.

**Comment by Universal-Omega** [comment]
This may be fixed by #88 but I have not tested it.

**Comment by Universal-Omega** [comment]
Can you please confirm the above worked for you if possible? I did verify it doesn't fatal, but I couldn't confirm much beyond that at the moment. Thanks!
